catppuccin_tmux.conf

```diff
@@ -137,7 +137,7 @@ set -wgF pane-border-style "#{E:@catppuccin_pane_border_style}"
 %elif "#{==:#{@catppuccin_window_status_style},rounded}"
 
   set -gq @catppuccin_window_left_separator "#[fg=#{@_ctp_status_bg},reverse]#[none]"
-  set -gq @catppuccin_window_middle_separator " "
```

Comment:
Why not use `-o`?

Reply:
lol I have [no idea](https://github.com/catppuccin/tmux/issues/462#issuecomment-2599580603) when that flag is supposed to be used...

I'll go ahead and add it here.

Reply:
Are you saying the `-o` flag can be used instead of the change I added?

Reply:
The `-o` flag is clearly documented: https://man7.org/linux/man-pages/man1/tmux.1.html#OPTIONS

Reply:
Why did you add `-F`?

Reply:
Forgot to remove it from the original draft.